src/_pytest/compat.py

```diff
@@ -327,8 +332,12 @@ def safe_getattr(object: Any, name: str, default: Any) -> Any:
     """
     try:
         return getattr(object, name, default)
-    except TEST_OUTCOME:
-        return default
+    except BaseException as exc:
+        from _pytest.outcomes import TEST_OUTCOME
```

Comment:
Just curious, why move the import?

Reply:
To resolve circular imports (see the commit message ;)).